contrib/imdocker/imdocker.c

```diff
@@ -64,7 +64,6 @@ extern int Debug;
 
 #define USE_MULTI_LINE
 //#define ENABLE_DEBUG_BYTE_BUFFER
```

Comment:
pls remove commented out code - you could change to #define ENABL... 0/1 and then check that value